gratipay/models/team/mixins/takes.py

```diff
@@ -3,6 +3,7 @@
 from collections import OrderedDict
 from decimal import Decimal as D
 
+from gettext import gettext as _
```

Comment:
Are you sure we import from `gettext`? I think we have some wrappers that we use instead of raw `gettext`, need to check ...